iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveTableUtil.java

```diff
@@ -302,7 +302,7 @@ static String getParseData(String parseData, String specId, ObjectMapper mapper,
 
   static JobConf getPartJobConf(Configuration confs, org.apache.hadoop.hive.ql.metadata.Table tbl) {
     JobConf job = new JobConf(confs);
-    job.set(ColumnProjectionUtils.READ_COLUMN_NAMES_CONF_STR, Constants.ICEBERG_PARTITION_TABLE_SCHEMA);
+    job.set(ColumnProjectionUtils.READ_COLUMN_NAMES_CONF_STR, "partition,spec_id");
```

Comment:
`ImmutableList.of("partition", "spec_id")` could we move to constants `PARTITION_COLUMNS`

Reply:
Fixed. Thanks.